plugin-server/src/main/ingestion-queues/batch-processing/each-batch-async-handlers.ts

```diff
@@ -7,23 +7,83 @@ import { runInstrumentedFunction } from '../../utils'
 import { KafkaJSIngestionConsumer } from '../kafka-queue'
 import { eachBatch } from './each-batch'
 
+// TODO: remove once we've migrated
 export async function eachMessageAsyncHandlers(message: KafkaMessage, queue: KafkaJSIngestionConsumer): Promise<void> {
     const clickHouseEvent = JSON.parse(message.value!.toString()) as RawClickHouseEvent
     const event = convertToIngestionEvent(clickHouseEvent)
 
+    await Promise.all([
+        runInstrumentedFunction({
+            server: queue.pluginsServer,
+            event: event,
+            func: () => queue.workerMethods.runAppsOnEventPipeline(event),
```

Comment:
tangential: This workerMethods dependency is a strange and confusing feature of the events ingestion imo. No action required though.

Reply:
would be nice to do a clean-up, but maybe we just wait until we switch to the batching world for exports and persons